torch/jit/_recursive.py

```diff
@@ -371,16 +371,22 @@ def init_fn(script_module):
             cpp_module.setattr(name, scripted)
             script_module._modules[name] = scripted
 
-        # 3. Copy @ignored/@unused methods from the original `nn_module` to the new ScriptModule.
+        # 3. Copy @ignored/@unused methods and properties from the original `nn_module` to the new ScriptModule.
         #    This ensures we can access these Python methods on the ScriptModule.
         for name in dir(nn_module):
             item = getattr(nn_module, name, None)
-            if not inspect.ismethod(item):
-                continue
-            if _jit_internal.is_ignored_fn(item):
+            if inspect.ismethod(item) and _jit_internal.is_ignored_fn(item):
                 unbound_function = getattr(type(nn_module), name)
                 bound_method = unbound_function.__get__(script_module)
                 setattr(script_module, name, bound_method)
+            elif isinstance(item, property) and _jit_internal.is_ignored_fn(item.fget):
+                unbound_getter = getattr(type(nn_module), item.fget.__name__)
```

Comment:
why are we doing anything if `_jit_internal.is_ignored_fn(item.fget` ?

Reply:
Don't we still need to attach the property to the scripted module? Just as we do with ignored functions?

Reply:
yea, good point